README.md

```diff
@@ -86,8 +86,8 @@ Like a global install of _any_ Node.js meta / task runner tool (e.g., `eslint`,
 ... so instead, we **strongly recommend** a local install described in the
 next section!
 
-To help you keep up with project-specific builder requirements, a globally-
-installed `builder` will detect if a locally-installed version of `builder` is
+To help you keep up with project-specific builder requirements, a globally-installed
```

Comment:
The line break was showing up as a space after the hyphen, like this:
![screen shot 2015-12-20 at 4 20 20 pm](https://cloud.githubusercontent.com/assets/6352327/11921100/e55d5878-a735-11e5-8433-c1378018502f.png)